storagenode/storagenodedb/vouchers.go

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storagenodedb
+
+import (
+	"context"
+	"database/sql"
+	"strings"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/storj"
+	"storj.io/storj/storagenode/vouchers"
+)
+
+type vouchersdb struct{ *InfoDB }
+
+// Vouchers returns database for storing vouchers
+func (db *DB) Vouchers() vouchers.DB { return db.info.Vouchers() }
+
+// Vouchers returns database for storing vouchers
+func (db *InfoDB) Vouchers() vouchers.DB { return &vouchersdb{db} }
```

Comment:
I'm not sure why it's wrapped like that, but all the other DBs in the storagenodedb package are like that. I just followed the precedent

Reply:
During the transition period, we had two sqlite databases hence we needed the wrapping to clearly separate those two. But we can now start removing the separate wrapping. It's fine either way here, because it'll be eventually fixed.